lib/engine/game/g_18_ms/game.rb

```diff
@@ -157,11 +157,7 @@ class Game < Game::Base
         ].freeze
 
         # Game will end after 10 ORs (or 11 in case of optional rule) - checked in end_now? below
```

Comment:
This comment is now out-of-date, there is no longer an `end_now?` method.
```suggestion
        # Game will end after 10 ORs (or 11 in case of optional rule)
```